packages/jbrowse-web/src/plugins/LinearGenomeView/components/ScaleBar.js

```diff
@@ -12,6 +12,7 @@ const styles = (/* theme */) => ({
     textAlign: 'left',
     width: '100%',
     position: 'relative',
+    userSelect: 'none',
```

Comment:
I'm guessing this is to prevent highlighting the ruler numbers when dragging the rubberband? It might be better to do this via `event.preventDefault()` instead, in case there are other less obvious side-effects that need to be stopped, too.

Reply:
I guess I'm uncertain what preventDefault does (or put differently, I don't know what the default it is preventing is) whereas this has a clear meaning to me. I could do some research into what preventDefault does in this case but that is just my first thought

Reply:
preventDefault just stops the click event from doing anything else besides what's in your method. Without it, the other DOM elements will also see the click/mousemove events and do what they normally would with them (like highlighting text, but could be other things, too).